pyintegration/test_helper/__init__.py

```diff
@@ -1,37 +1,57 @@
 #
-# Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
 #
-""" Utilities for starting the Deephaven JVM. """
+""" This module supports bootstrapping a Deephaven Python Script session from Python."""
 
 import os
 import re
 import warnings
 from glob import glob
-from os import path
 
 import jpy
 import jpyutil
 
 
-# from deephaven import initialize
-# from .conversion_utils import _isStr
-
-
-def start_jvm(devroot=None,
-              workspace=None,
-              propfile=None,
-              keyfile=None,
-              verbose=False,
-              skip_default_classpath=None,
-              # The following are the jpyutil.init_jvm options which are passed through after attaching our options
-              java_home=None,
-              jvm_dll=None,
-              jvm_maxmem=None,
-              jvm_classpath=None,
-              jvm_properties=None,
-              jvm_options=None,
-              config_file=None,
-              config=None):
+DEFAULT_DEVROOT = os.environ.get('DEEPHAVEN_DEVROOT', "/tmp/pyintegration")
```

Comment:
@jmao-denver via call requests that these change to current dir for non-docker testing